tests/unit/dataset/dataset_definition_test.py

```diff
@@ -385,24 +418,12 @@ def test_check_equality_of_load_from_yaml_and_load_from_dictionary_dump(tmp_path
             id='empty_resources_dict',
         ),
 
-        pytest.param(
-            1,
-            False,
-            id='int_resources',
-        ),
```

Comment:
this test case is not supported anymore but it also doesn't make sense to expect something valid from the input (`DatasetDefinition.resources == 1`)